rpcs3/Emu/Cell/lv2/sys_vm.cpp

```diff
@@ -48,13 +48,18 @@ sys_vm_t::sys_vm_t(utils::serial& ar)
 	g_fxo->get<sys_vm_global_t>().total_vsize += size;
 }
 
-error_code sys_vm_memory_map(ppu_thread& ppu, u32 vsize, u32 psize, u32 cid, u64 flag, u64 policy, vm::ptr<u32> addr)
+error_code sys_vm_memory_map(ppu_thread& ppu, u64 vsize, u64 psize, u32 cid, u64 flag, u64 policy, vm::ptr<u32> addr)
 {
 	ppu.state += cpu_flag::wait;
 
 	sys_vm.warning("sys_vm_memory_map(vsize=0x%x, psize=0x%x, cid=0x%x, flags=0x%x, policy=0x%x, addr=*0x%x)", vsize, psize, cid, flag, policy, addr);
 
-	if (!vsize || !psize || vsize % 0x2000000 || vsize > 0x10000000 || psize > 0x10000000 || policy != SYS_VM_POLICY_AUTO_RECOMMENDED)
+	if (!vsize || !psize || vsize % 0x200'0000 || vsize > 0x1000'0000 || psize > 0x1000'0000 || pize < 0x10'0000 || psize % policy != SYS_VM_POLICY_AUTO_RECOMMENDED)
```

Comment:
Hello, shouldn't be:
```suggestion
	if (!vsize || !psize || vsize % 0x200'0000 || vsize > 0x1000'0000 || psize > 0x1000'0000 || psize < 0x10'0000 || psize % policy != SYS_VM_POLICY_AUTO_RECOMMENDED)
```